boards/arm/nucleo_f756zg/nucleo_f756zg.dts

```diff
@@ -53,16 +54,22 @@
 };
 
 &usart2 {
+	pinctrl-0 = <&usart2_tx_pd5
+		     &usart2_rx_pd6
+		     &usart2_rts_pd4
+		     &usart2_cts_pd3>;
```

Comment:
Just a suggestion, but for readability (highly subjective) could have been:
```suggestion
	pinctrl-0 = <&usart2_tx_pd5 &usart2_rx_pd6
		     &usart2_rts_pd4 &usart2_cts_pd3>;
```

Reply:
Agreed, this looks better, will apply this. If I remembre correctly, some other boards have same definitions, I will update them too.

Reply:
So, nucleo_f746zg is another board that will receive the same refactoring

Reply:
Done in latest push